client/slack.go

```diff
@@ -147,6 +147,20 @@ func (sl *slackClient) handleMessageEvents() {
 				continue
 			}
 			sl.incomingMessages <- toFlyteMessageEvent(v, u)
+
+		case *slack.ReactionAddedEvent:
+			u, err := sl.client.GetUserInfo(v.User)
+			log.Debug().Msgf("received reaction event for type = %v", v)
+			if err != nil {
+				log.Err(err).Msgf("cannot get info about user=%s: %v", v.User, err)
+				continue
+			}
+			itemuser, err := sl.client.GetUserInfo(v.ItemUser)
+			if err != nil {
+				log.Err(err).Msgf("cannot get info about item user=%v: %v", v.ItemUser, err)
+				continue
```

Comment:
```suggestion
```

Reply:
Hi Paul,  If any error has occurred then we'd require to not process this workflow.
As , payload sends the user and itemuser -- line no 227 filename: slack.go 

	return reactionEvent{
		Type:     e.Type,
		User:     newUser(u),
		ItemUser: newUser(itemUser),
		Item: ReactionItem{
			Type:      e.Item.Type,
			Channel:   e.Item.Channel,
			Timestamp: e.Item.Timestamp,
		},
		Reaction:       e.Reaction,
		EventTimestamp: e.EventTimestamp,
	}

Reply:
keep in mind `GetUserInfo` can have high failure rate

Reply:
Yes,  I will keep this continue statement for now, If we get lot of issues then I will further enhance or take it up.